### ISSUE TITLE
Typo Error: `dim` Parameter in `MeshAutoencoder`

### ISSUE THREAD

**Issue by qixuema** [lucidrains/meshgpt-pytorch#33]
Hi Phil,

I've noticed a potential typo error. A few days ago, you removed the `dim` parameter in the `MeshAutoencoder`, but I still see it being used [here](https://github.com/lucidrains/meshgpt-pytorch/blob/378586713fe599baccfdaa68bd25ee5844a432eb/meshgpt_pytorch/meshgpt_pytorch.py#L580). Is this a typo error?

```python
for _ in range(local_attn_encoder_depth):
    self.encoder_local_attn_blocks.append(nn.ModuleList([
        LocalMHA(dim = dim, **attn_kwargs, **local_attn_kwargs),
        nn.Sequential(RMSNorm(curr_dim), FeedForward(curr_dim, glu = True, dropout = ff_dropout))
    ]))
```

Best regards,
Xueqi

**Comment by lucidrains** [comment]
@qixuema yes, my bad, thank you
